Dockerfile

```diff
@@ -551,6 +551,7 @@ ARG TARGETPLATFORM
 RUN --mount=type=cache,sharing=locked,id=moby-build-aptlib,target=/var/lib/apt \
     --mount=type=cache,sharing=locked,id=moby-build-aptcache,target=/var/cache/apt \
         xx-apt-get install --no-install-recommends -y \
+            dpkg-dev \
```

Comment:
Why do we now need `dpkg-dev`?

Reply:
https://github.com/moby/moby/pull/44812#issuecomment-1380699254

Reply:
Ah, you pushed as I was reviewing.